### ISSUE TITLE
A11y_SQLAzureDataStudio_ConnectToASQLServer_AdvancedProperties_AI4D: Control type assigned to the drop down field is incorrect.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9206]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#Win32;#AI4D;#Benchmark;#A11y_SQLAzureDataStudioBenchmark;#A11ySev2;#MAS4.1.2;#MAS1.3.1;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader: NVDA(2019.3.1), JAWS(2020.2001.70)
MAS reference: MAS4.1.2,MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Connection' icon.
4. Navigate till Advanced button under Connection details & hit enter.
5. Launch Accessibility Insights for Desktop.
6. Check if control type assigned is correct for all drop down menus.

**Actual:**
The control type assigned to the combo box is incorrect which is assigned as 'toolbar'.

**Expected:**
The control type should be changed from 'toolbar' to 'button' for the combo box present.

**User Impact:**
Screen reader users might get confused as the announcement made by screen reader for the control is 'toolbar'.
Also the combo box button present here performs functionality of expanding and collapsing the menu items below it and nothing else.
Therefore its control type should be assigned as button and not toolbar.

**Attachment link reference:**
[9206_A11y_SQLAzureDataStudio_ConnectToASQLServer_AdvancedProperties_AI4D.zip](https://github.com/microsoft/azuredatastudio/files/4291547/9206_A11y_SQLAzureDataStudio_ConnectToASQLServer_AdvancedProperties_AI4D.zip)

**Comment by alanrenmsft** [comment]
should be a button role with collapsed state

**Comment by Charles-Gagnon** [comment]
This is the case for all bars like this in VS Code - let's figure out if they got an exception for this or something before making the change ourselves.

**Comment by Harika1702** [comment]
<img width="374" alt="image" src="https://user-images.githubusercontent.com/100270074/219293936-ca73abbc-e63f-4e93-bb45-ba180a5ecf87.png">
Verified this issue now issue is fixed.
Version: 1.41.1 (user setup)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:46:39.274Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22623
GitHub Tags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
